### PR TITLE
Show `tokenError` for any options errors (not just 404).

### PR DESCRIPTION
**Pull request by maxpeterson**
`django-user-management` was returning a `400` rather than a `404` when the token was not valid. This resulted in the error being ignored.

This appears to relate to https://github.com/incuna/django-user-management/pull/99 (cc @KevinEtienne )

I could have changed the check for a `404` to a `400` but it seems like any error from the token endpoint should probably trigger the `tokenError`. (cc @grahamgilchrist )

### CODE REVIEW

**Comment by maxpeterson** [comment]
@incuna/js please review / merge

**Comment by KevinEtienne** [comment]
I think we needed to differentiate an expired token from a non existing one. I don't remember for which project we did that however.

**Comment by KevinEtienne** [comment]
Related in part to https://github.com/incuna/abbvie-humira-epatient/issues/1287

File: src/password/scripts/directives.js [hunk 1]
```diff
@@ -91,7 +91,7 @@
                             scope.fields = response.data.actions.PUT;
                         }, function (response) {
                             // If the response returns a 404 and we have a token.
```
Comment (wytrych) [inline src/password/scripts/directives.js]: This comment is outdated now

**Comment by maxpeterson** [comment]
@wytrych updated the comment.

**Pull request merged**